src/components/NavBar.tsx

```diff
@@ -1,50 +1,71 @@
-import * as React from 'react'
+// @refresh reset
 import AppBar from '@mui/material/AppBar'
 import Button from '@mui/material/Button'
 import { grey } from '@mui/material/colors'
 import Toolbar from '@mui/material/Toolbar'
 import Typography from '@mui/material/Typography'
-import Link from '../Link'
```

Comment:
Can you make use of the link component instead removing it? This would allow client-side navigation. See [here](https://nextjs.org/docs/api-reference/next/link) and [here](https://nextjs.org/learn/basics/navigate-between-pages/link-component)